index.html

```diff
@@ -386,7 +385,17 @@ <h4>Octane</h4>
       </section>
 
       <section aria-labelledby="section-element-modifiers">
-        <h3 id="section-element-modifiers">Use element modifiers instead of didInsertElement</h3>
+        <h3 id="section-element-modifiers">Element Modifiers</h3>
+        <p>There are three new element modifiers {{did-insert}}, {{did-update}}, and {{will-destroy}} that you may use to replace the lifecycle hooks didInsertElement, willDestroy, and didUpdate.
+        </p>
+        <p>
+          In order for these modifiers to work, you need to install the <a href="https://github.com/emberjs/ember-render-modifiers" target="_blank">@ember/render-modifiers package</a>.
+          <pre data-src="code/render-modifiers/render-modifiers.handlebars"></pre>
```

Comment:
```suggestion
          <pre data-src="code/render-modifiers/render-modifiers.shell"></pre>
```

If you rename the file to end in `.shell`, it will have better syntax highlighting.